### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by tdwcks**

### CODE REVIEW

File: app/views/sprint-7/prototype-1/guidance/risk-1.html [hunk 1]
```diff
@@ -65,13 +65,16 @@ <h2 class="govuk-heading-m" id="subsection-title">References</h2>
             <ul class="govuk-list govuk-!-font-size-16">
               <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
```
Comment (UsmanAfzal) [inline app/views/sprint-7/prototype-1/guidance/risk-1.html:66]: Something I missed but horizontal rules are not allowed as a child element of an unordered list. These need to be removed. Also the headings are not necessary, links are sufficient and would also follow the pattern of gov.uk https://www.gov.uk/guidance/information-requirements-for-pension-schemes-the-basics#relief-at-source-annual-return-of-information.

File: app/routes.js [hunk 2]
```diff
@@ -1264,7 +1264,7 @@ router.get("/:sprint/prototype/:pathWay", (req, res) => {
     // generate the question url
 
     let question_url = "/"+req.params.sprint+"/prototype/"+pathway_key+"/question/"+this_row.id;
-    let view_link = "<a href='"+question_url+"'>View</a>";
+    let view_link = "<a href='"+question_url+"'>View <span class='govuk-visually-hidden'>"+ this_row.label+"</span></a>";
```
Comment (UsmanAfzal) [inline app/routes.js:1267]: I suggest adding this instead `aria-label='View control - "+this_row.label+"'`

**Review by UsmanAfzal — CHANGES_REQUESTED** [review]
@tdwcks I've added one comment related to the references area  on the risk page.

File: app/views/sprint-7/prototype-1/guidance/risk-1.html (hunk 1 above)
Comment (tdwcks) [reply app/views/sprint-7/prototype-1/guidance/risk-1.html]: Updated thanks @UsmanAfzal

File: app/routes.js (hunk 2 above)
Comment (UsmanAfzal) [reply app/routes.js:1267]: @tom To make this more clear can we add "control" before the label so it reads "View control - [label]"

`<span class='govuk-visually-hidden'>control - "+ this_row.label+"</span>`

**Review by UsmanAfzal — APPROVED** [review]

**Review by UsmanAfzal — APPROVED** [review]

**Pull request merged**